plugins/node/opentelemetry-instrumentation-memcached/src/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InstrumentationConfig as BaseInstrumentationConfig } from '@opentelemetry/instrumentation';
+
+export interface InstrumentationConfig extends BaseInstrumentationConfig {
+  collectCommand?: boolean;
```

Comment:
Could you use `enhancedDatabaseReporting` like other db instrumentation do (ex: [mongodb](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/621f0141e406b75ef11a77fb85ce8caab9c8a272/plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts#L25)) so its consistent ?

Reply:
The name `enhancedDatabaseReporting`

1. is not descriptive by itself and;
2. sounds like it can affect multiple aspects of how the instrumentation works.

I'd personally stay away from such configuration options anywhere.
Happy to change it for you if that's a breaker though! :)

Reply:
We inherited this from Opencensus, i don't mind changing it but i would prefer to make it the same for all instrumentation anyway. Could you use `enhancedDatabaseReporting` and open an issue to discuss this ?

Reply:
Could you please share a reference to that in OpenCensus?

Reply:
> Could you please share a reference to that in OpenCensus?

I was wrong that wasn't from Opencensus directly but from the google cloud tracer sdk (https://github.com/googleapis/cloud-trace-nodejs/search?q=enhancedDatabaseReporting. 
You can lookup the initial proposal here: https://github.com/open-telemetry/opentelemetry-js/issues/214

> Since many of the auto-instrumentations will be provided by 3rd parties we can never ensure there is a finite list of options used across all instrumentations across all libs.

We cannot ensure consistency with 3rd party instrumentations but we can at least for 1st party instrumentation.

There was a discussion at the time here about this: https://github.com/open-telemetry/opentelemetry-js/issues/614

Reply:
Thanks for the info, @vmarchaud! :pray: I renamed the config option.

Reply:
I join the voice of not using `enhancedDatabaseReporting` parameter. Would be happy to see it replaced with meaningful names everywhere